stat.go

```diff
@@ -648,6 +648,71 @@ func KullbackLeibler(p, q []float64) float64 {
 	return kl
 }
 
+// OrdinaryLinearRegression computes the simple linear regression of x and y.
```

Comment:
Do we really need "Ordinary"? Ordinary is in general implied, it's the non-ordinary ones that need an additional moniker.